manageiq-operator/Makefile

```diff
@@ -88,7 +88,8 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manageiq-operator crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	sed -i "/creationTimestamp\|namespace/d" config/rbac/role.yaml
```

Comment:
This is going to fail in MacOS.  I'll show you offline, but the fix is obnoxious.

Reply:
Fixed